lib/__tests__/uploadFolder.ts

```diff
@@ -164,6 +163,7 @@ describe('lib/cms/uploadFolder', () => {
     });
 
     it('deletes the temporary directory', async () => {
+      createTmpDirSync.mockReturnValue('folder');
```

Comment:
The test before this was setting up this mock, so this started failing when `clearMocks` was set to true.